### DISCUSSION TITLE
AttributeError: 'FPDF' object has no attribute 'set_page_background'

### DISCUSSION THREAD

**Discussion by Tilchy** [py-pdf/fpdf2#517] [Q&A]
Hi as the Title already states I get Attribute Error.. I'm using python 3.10 

![image](https://user-images.githubusercontent.com/80243427/188103324-71e1c854-5d1b-4f64-8dca-a0c936c495cb.png)

**Comment by Tilchy** [comment]
What fixed it for me was:
1. Python 3.9
2. pip install git+https://github.com/PyFPDF/fpdf2.git@master

¯\_(ツ)_/¯

**Reply by semaeostomea** [reply]
Hey,
you didn't have to downgrade to python 3.9, it works just fine with 3.10

You probably had fpdf2 installed through pypi before. The `.set_page_background()` function is part of version 2.5.7, which hasn't been released on pypi yet. This is why installing from the git repository fixes it 👍

**Comment by Lucas-C** [comment]
I confirm @semaeostomea answer.
For reference, here is the original PR: https://github.com/PyFPDF/fpdf2/pull/497
And the library release notes, where you can check what hasn't been released yet: [CHANGELOG.md](https://github.com/PyFPDF/fpdf2/blob/master/CHANGELOG.md)
